nevergrad/benchmark/experiments.py

```diff
@@ -1129,6 +1131,229 @@ def rocket(seed: tp.Optional[int] = None, seq: bool = False) -> tp.Iterator[Expe
 
 
 @registry.register
+def ng_full_gym(
+    seed: tp.Optional[int] = None,
+    randomized: bool = True,
+    multi: bool = False,
+    big: bool = False,
+    memory: bool = False,
+    ng_gym: bool = False,
+    conformant: bool = False,
+) -> tp.Iterator[Experiment]:
+    """Gym simulator. Maximize reward.
+    Many distinct problems."""
+    env_names = GymMulti.env_names
+    if ng_gym:
+        env_names = GymMulti.ng_gym
+    seedg = create_seed_generator(seed)
+    optims = get_optimizers("basics", "progressive", "splitters", "baselines", seed=next(seedg))
+    optims += ["DiagonalCMA"]
+    optims += get_optimizers("split_noisy", seed=next(seedg))
+    optims += ["TBPSA"]
+    optims += ["NGOpt"]
+    optims = ["NGOpt", "PSO", "DiscreteOnePlusOne", "DE", "CMandAS2"] + [  # type: ignore
+        o for o in optims if "ProgD13" in str(o) or "ProgInf" in str(o)  # type: ignore
+    ]
+    if multi:
+        controls = ["multi_neural"]
+    else:
+        controls = (
+            [
+                "neural",
+                "structured_neural",
+                "memory_neural",
+                "stackingmemory_neural",
+                # "noisy_neural",
+                # "noisy_scrambled_neural",
+                # "scrambled_neural",
+                "deep_neural",
+                "semideep_neural",
+                # "linear",
+            ]
+            if not big
+            else ["neural"]
+        )
+    if memory:
+        controls = ["stackingmemory_neural", "deep_stackingmemory_neural", "semideep_stackingmemory_neural"]
+        controls += ["memory_neural", "deep_memory_neural", "semideep_memory_neural"]
+        controls += [
+            "extrapolatestackingmemory_neural",
+            "deep_extrapolatestackingmemory_neural",
+            "semideep_extrapolatestackingmemory_neural",
+        ]
+        assert not multi
+    if conformant:
+        controls = ["stochastic_conformant"]
+    controls = ["neural"]
```

Comment:
That's weird you're defining a lot of controls then setting controls to ["neural"] afterwards. 
Do you want to add "neurals" instead or are you missing a elif above and a else there ?

Reply:
Removed, this was just a first hack for running additional xps in a single case.